plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioOrcConnectorTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+import io.trino.Session;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.iceberg.IcebergFileFormat.ORC;
+import static io.trino.plugin.iceberg.IcebergTestUtil.orcSupportsIcebergFileStatistics;
+import static io.trino.plugin.iceberg.IcebergTestUtil.orcSupportsRowGroupStatistics;
+import static io.trino.plugin.iceberg.IcebergTestUtil.orcWithSmallRowGroups;
+
+// Due to fact some tests running on MinIO are memory consuming, we want to prevent running them in parallel.
+// Project's default for air.test.parallel is 'methods'. So singleThreaded is set to true. As described in TestNG docs.
+// If set to true, all the methods on this test class are guaranteed to run in the same thread even if the tests
+// are currently being run with parallel="methods". Similar as when 'air.test.parallel=classes' is set
```

Comment:
The first line
```
// Due to fact some tests running on MinIO are memory consuming, we want to prevent running them in parallel.
```

is useful, thanks for adding it.
The other 3 are not specific to this class. We don't need to document the framework (testng) behavior every time we use `@Test(singleThreaded = true)`.

BTW does it still apply after you reduced data sets size in tests?

Reply:
I left them in case `<air.test.parallel>classes</air.test.parallel>` will be removed at some point.
Then we still want those methods to not run in parallel as single method usage may  reach 1,5GB which disallows running them in parallel to avoid flakiness.

Reply:
Actually after reducing data sets i will see if this is still required here.